### DISCUSSION TITLE
Is it possible to play and loop only a selection of patterns ?

### DISCUSSION THREAD

**Discussion by Maojojo** [hydrogen-music/hydrogen#1159] [General]
Hello everyone,

is it possible to play and loop only a selection of patterns ?

Regards,
Maojojo

**Comment by oddtime** [comment]
If you mean a selection of patterns to be played at same time yes, with stack mode.
Or do you mean a sequence?

**Reply by Maojojo** [reply]
I mean in a sequence. Imagine I have a full song with succession of 15 patterns, but I want to loop number 2,3 and 4 for practice. Like song mode when playing but for selected moment in song, from frame 2 to 4. Am I clear ?

**Reply by oddtime** [reply]
Yes you are clear. It would be nice to have this feature.
As far as I know it is not possible currently. There is already an open issue for it #384
